doc/source/user_guide/meshing_workflow/index.rst

```diff
@@ -0,0 +1,16 @@
+.. _ref_meshing_workflow:
+
+================
+Meshing Workflow
+================
+Fluent meshing guided workflows are designed to make it easier to generate a
+volume mesh starting from CAD geometries. You can use PyFluent to access the guided workflows.
+
+Workflows are divided into the new :ref:`ref_user_guide_new_meshing_workflows` and :ref:`ref_user_guide_meshing_workflows` which align with the journaled syntax.
```

Comment:
Vale doesn't like _journaled_, but _journal_ is more appropriate here, anyway.

Reply:
As mentioned, the comma before _and_ would ensure that the clause after which is only bound to the classic part.